### DISCUSSION TITLE
图书馆提示图序和图题，加黑；表序和表题，加黑

### DISCUSSION THREAD

**Discussion by tx19980520** [sjtug/SJTUThesis#681] [提问] [answered]
图序和图题，加黑；表序和表题，加黑，图书馆专家审核的反馈，请问这个部分有相应的update或暂时的解决方案吗

**Comment by skyzh** [comment, marked as answer]
master 分支已经解决了这个问题。如果您不想更新模版，也可以参考 https://github.com/sjtug/SJTUThesis/discussions/623

**Reply by izackwu** [reply]
@skyzh master 分支的「原创性声明」是新版的，不太符合目前的本科毕设要求，所以在 rebase 到 master 分支之后，还需要 revert 关于原创性声明的那一个 commit
